### ISSUE TITLE
Can we use hummusRecipe to encrypt the PDF in ReactJS ?

### ISSUE THREAD

**Issue by sagar1987** [chunyenHuang/hummusRecipe#200]
I tried to install npm i hummus-recipe --save in ReactJs application , I am getting below error.

./node_modules/muhammara/node_modules/node-pre-gyp/lib/info.js
Module not found: Can't resolve 'aws-sdk' in 'C:\ReactPDFDownload\decision_tree_ui\decision_tree_ui\node_modules\muhammara\node_modules\node-pre-gyp\lib'

**Comment by chunyenHuang** [comment]
I dont think so. HummusJS can only be used in the Node.js system not in the Browser.
